.storybook/config.js

```diff
@@ -1,35 +1,55 @@
-import * as React from 'react'
-import { configure, addDecorator } from '@storybook/react'
-import { withScreenshot } from 'storycap'
-import { createGlobalStyle } from 'styled-components'
-import theme from '../src/theme'
-import {Provider} from '../src/index'
+import * as React from "react";
+import { configure, addDecorator } from "@storybook/react";
+import { useDarkMode } from "storybook-dark-mode";
+import { withScreenshot } from "storycap";
+import { createGlobalStyle } from "styled-components";
+import theme from "../src/theme";
+import { Provider } from "../src/index";
 
 const GlobalStyle = createGlobalStyle([], {
-  '*': {
-    boxSizing: 'border-box'
+  "*": {
+    boxSizing: "border-box",
   },
   body: {
     lineHeight: 1.5,
     margin: 0,
     fontFamily: theme.font,
-    background: 'white',
-    webkitFontSmoothing: 'antialiased'
-  }
-})
+    webkitFontSmoothing: "antialiased",
+  },
+});
+
+const ThemeProvider = ({ children }) => {
+  const isDark = useDarkMode();
 
-function withGlobalStyles (storyFn) {
   return (
     <React.Fragment>
       <GlobalStyle />
-      <Provider>
-        {storyFn()}
+      <Provider
+        theme={
+          isDark
+            ? {
+                colors: {
```

Comment:
This is just temporary, we'll need to define how the theme format will look like and define both light and dark in `theme.ts`